src-tauri/src/manager.rs

```diff
@@ -111,17 +119,36 @@ fn start_watcher(name: &'static str, tx: Sender<WatcherMessage>) {
         })
         .unwrap();
 
-        // Wait for the child to exit
-        let output = child
-            .unwrap()
-            .wait_with_output()
-            .expect("failed to wait on child");
-
-        // Send the process output to the manager
-        tx.send(WatcherMessage::Stopped {
-            name: name.to_string(),
-            output,
-        })
-        .unwrap();
+        loop {
```

Comment:
This loop doesn't loop clean to me, what are you looping over? When does the looping end?

Reply:
This loops for the entirety of the program, checks whether the child process is still alive. If it is not sends the output to handle (there was a regression here due to me fighting the borrow checker) and breaks from the loop. The other arm of the match is executed every loop the process is alive. checks if the variable alive is still true. If not kills the process and exits the loop.